<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>gframe m4k'r</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>192</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>192</width>
      <height>192</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="formatOpt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>198</y>
      <width>78</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>output format</string>
    </property>
    <item>
     <property name="text">
      <string>Gfm 4b</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gfm 32b</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>raw 4b</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>raw 8b</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>raw 16b</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>raw 32b</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="transparentOpt">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>220</y>
      <width>78</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>transparency color</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>not transparent</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>up left pixel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>down left pixel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>pink</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>white</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="frameNbInput">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>240</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QLabel" name="framesNfo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>238</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>frame</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="horizontalCheck">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>240</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>horizontal frameset</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="convertBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>0</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
